management/rate_limits.go

```diff
@@ -0,0 +1,42 @@
+package management
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+type RateLimitError struct {
+	error
+	RateLimit
+}
+
+type RateLimit struct {
+	Limit     uint64
+	Remaining uint64
+	Reset     time.Time
+}
+
+func newRateLimit(h http.Header) (*RateLimit, error) {
+	reset, err := strconv.Atoi(h.Get(RateLimitReset))
+	if err != nil {
+		return nil, fmt.Errorf("error parsing x-ratelimit-reset header value: %s", err)
```

Comment:
minor, but i would use this instead:

`return nil, fmt.Errorf("error parsing %s header value: %s", RateLimitReset, err0